dbbackup/tests/test_connectors.py

```diff
@@ -1,7 +1,8 @@
 from __future__ import unicode_literals
 
 import os
-from mock import patch, mock_open
+from django.db import DEFAULT_DB_ALIAS
+from mock import patch, mock_open, MagicMock
```

Comment:
Could you only use `mock.patch` like made here: https://github.com/MiiRaGe/django-dbbackup/blob/d33f8590f691dae46d19c31239dd546b7b8249bd/dbbackup/tests/test_connectors.py#L145